src/lib.rs

```diff
@@ -93,7 +93,7 @@
 //!
 #![forbid(unsafe_code, future_incompatible, rust_2018_idioms)]
 #![deny(missing_debug_implementations, nonstandard_style)]
-#![warn(missing_docs, missing_doc_code_examples, unreachable_pub)]
+#![warn(missing_docs, rustdoc::missing_doc_code_examples, unreachable_pub)]
```

Comment:
unrelated change. If needed, please split it into a separate commit (not PR) with its own rationale.

Reply:
Done + Replaced MR with one commit for history stay short.

Reply:
> Done

:+1: 

> + Replaced MR with one commit for history stay short.

:( AFAIK there are no benefits of keeping history short but atomic commits do (and hence the reason all pre-GH projects follow that strictly).